### PR TITLE
TPC: Prepare dead channel map for tracking, misc updates

### PR DESCRIPTION
**Pull request by wiechula**

### CODE REVIEW

**Comment by wiechula** [comment]
@davidrohr , this prepares the dead channel map for tracking.
It is transported via the `CalibdEdxContainer` class in the gpu calibration values. You can use the function
https://github.com/AliceO2Group/AliceO2/pull/12208/files#diff-c78132314df45bc20f548e08a118ae961dd2cc27c0dd21da135486940ed37490R128
to check if the pad is flagged dead. The dead status is store in the newly created class [GPU/GPUTracking/DataTypes/TPCPadBitMap.h](https://github.com/AliceO2Group/AliceO2/pull/12208/files#diff-382705ba0b6c406516cd6b627e084955080ed814bb9d93303e1665c353219c92R1) with dense bit packing.

File: GPU/GPUTracking/DataTypes/CalibdEdxContainer.h [hunk 1]
```diff
@@ -224,14 +233,17 @@ class CalibdEdxContainer : public o2::gpu::FlatObject
...
-  CalibsdEdx mCalibsLoad{CalibsdEdx::CalTopologyPol | CalibsdEdx::CalThresholdMap | CalibsdEdx::CalGainMap | CalibsdEdx::CalResidualGainMap | CalibsdEdx::CalTimeGain}; ///< flags to set which corrections will be loaded from the CCDB and used during calculation of the dE/dx
+  CalibdEdxTrackTopologySpline* mCalibTrackTopologySpline{nullptr}; ///< calibration for the track topology correction (splines)
+  CalibdEdxTrackTopologyPol* mCalibTrackTopologyPol{nullptr};       ///< calibration for the track topology correction (polynomial)
+  o2::gpu::TPCPadGainCalib mThresholdMap{};                         ///< calibration object containing the zero supression threshold map
+  o2::gpu::TPCPadGainCalib mGainMap{};                              ///< calibration object containing the gain map
+  o2::gpu::TPCPadGainCalib mGainMapResidual{};                      ///< calibration object containing the residual gain map
+  o2::gpu::TPCPadBitMap mDeadChannelMap{};                          ///< calibration object containing the residual gain map
```
Comment (davidrohr) [inline GPU/GPUTracking/DataTypes/CalibdEdxContainer.h:241]: I assume you need to increase the CLassDef of this class if you add a member?

**Review by davidrohr — COMMENTED** [review]
Looks good to me, I only think the ClassDef must be incremented.

File: GPU/GPUTracking/DataTypes/CalibdEdxContainer.h (hunk 1 above)
Comment (wiechula) [reply GPU/GPUTracking/DataTypes/CalibdEdxContainer.h:241]: Thanks, I overlooked it, indeed.

**Review by davidrohr — DISMISSED** [review]

**Pull request merged**